src/harness/testcases/registration_testcase.py

```diff
@@ -147,3 +147,69 @@ def test_10_3_4_2_5_3(self):
     # Check registration response
     self.assertFalse('cbsdId' in response)
     self.assertEqual(response['response']['responseCode'], 103)
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_REG_26(self):
+    """Category Error in Array request (responseCode 202)
+
+    The response should be SUCCESS for the first CBSD,
+    FAILURE for the second, third, and fourth CBSDs.
+    """
+
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Set up unique FCC IDs
```

Comment:
I'm not sure either. It seems other test cases use the device_x.json files when registering CBSDs. I'm fine with renaming to registration_x.json or keeping device_x.json. 

How about naming the devices by order needed in test case order for now?
device_a.json: cat A (reg.1)
device_b.json: cat B (reg.2)
device_c.json: cat A (reg.14)
device_d.json: cat B (reg.14)
device_e.json: cat A (reg.15)
device_f.json: cat A (reg.15)
device_g.json: cat A (reg.15)